text/0685-new-browser-support-policy.md

```diff
@@ -0,0 +1,286 @@
+matrix for Ember users.
+
+### Evergreen browsers
+
+Ember and Ember Data will support **all** versions of evergreen browsers which
+meet either of the following criteria:
+
+1. It is the latest version of the browser.
+2. It has at least **0.25%** marketshare usage, based on
+   [statcounter](https://gs.statcounter.com/).
+
+This policy will allow Ember to generally support the most recent 2-3 versions
+of the browser, and account for versions which may become temporary transition
+points. For example, when MS Edge made the switch to the new Chromium-backed
+version, the last non-Chromium version saw higher usage for a slightly longer
+period of time as the update process was not as standard, and required an
+explicit opt-in. These transitionary periods are important, but generally
+speaking temporary, and once the usage drops below the threshold Ember can drop
+support at that point.
+
+The supported versions of browsers are determined at the time of each _minor_
+release of Ember and Ember Data. In the release blog post, we will specify
+which versions of browsers are supported for this minor version, to keep users
+informed as usage changes.
+
+_Within_ a minor version, support works as follows:
+
+1. Support for a given browser version is _never_ removed.
+2. Support is _added_ for any new versions of a browser that have been released
+   while the minor is still supported.
+
+So for instance, if we were to release Ember 4.4 with support for Chrome 92, 93,
+and 94, then bugfixes and security fixes for those versions of Chrome will be
+supported until 4.4 is no longer an LTS, following the LTS support matrix. In
+addition, if Chrome were to release version 95 while 4.4 is still actively
+supported, then 4.4 will also support Chrome 95.
+
+### Non-evergreen browsers
+
+Ember and Ember Data will support the following versions of non-evergreen
+browsers:
+
+- Desktop
+  - Safari: 14
+- Mobile
+  - Safari: 14
+
+These versions will continue to be supported until support is explicitly dropped
+via a new RFC, and dropping support for any version of these browsers will
+require a new **major** release.
+
+In addition, all new major releases of these browsers will be supported as they
+are released. Like with evergreen browsers, support is retroactively added to
+any minor version of Ember which is still supported.
+
+#### Note on the versions chosen
+
+The versions chosen here are the latest versions of Safari, which having just
+been released this year, do not have 100% adoption. As such, there are still a
+significant number of users who are using previous versions of Safari. However,
+by the time this support policy is enacted and a new major version of Ember is
+released, this number will likely have shifted significantly.
+
+In addition, since Ember will not drop support for any version of Safari until
+the next major release after this policy is enacted (v5), this decision strikes
```

Comment:
RE: `v5`. 

1. My knowledge might be a little outdated or perhaps this is still up in the air.  Will ember versions >= `v4.0` remove IE support?  Is this the impetus for having this new browser support policy?

2. To clarify this comment about `v5`, are we stating that Safari version 13.1 and under will be supported for all of v4?  

If I have this right, then when looking across multiple versions of Ember, effectively `v4.0` removes IE support and `v5.0` removes Safari < 14 (but open to other removals depending on the landscape at the time).

Reply:
Correct, v4 will implement this new policy, this is noted a bit later but it is not super prominent, might be good to make it more prominent and note it higher up.

Re: v5, the note here is assuming that we would remove support in v5, but technically that is just the _earliest_ we could remove support, since this policy comes into effect in v4. If we were to release v5 without any new RFCs in the meantime that alter the policy, then this policy would remain in effect, and we would still support Safari 14 in v5.

That said, I think it’s very likely we will as a community want to drop support for some Safari versions whenever a major version happens, so I think its safe to assume it will be done.